src/storage/table/row_group.cpp

```diff
@@ -401,6 +401,15 @@ void RowGroup::NextVector(CollectionScanState &state) {
 	}
 }
 
+static FilterPropagateResult CheckRowIdFilter(TableFilter &filter, idx_t beg_row, idx_t end_row) {
+	// RowId columns dont have a zonemap, but we can trivially create stats to check the filter against.
+	BaseStatistics dummy_stats = NumericStats::CreateEmpty(LogicalType::ROW_TYPE);
+	NumericStats::SetMin(dummy_stats, UnsafeNumericCast<row_t>(beg_row));
```

Comment:
Can we use `NumericStats::Update` here?

Reply:
Unfortunately not, we can't call this directly cause we can't access NumericStatsData, and we can't call BaseStatistics.UpdateNumericStats because it doesn't set the `has_min` and `has_max` fields.